statistics/testdata/stats_suite_in.json

```diff
@@ -5,6 +5,47 @@
       "explain select * from t where a = 1 and b = 5 and 1 = 1"
     ]
   },
+  {
+    "name": "TestStatsVer2",
+    "cases": [
+      "explain select * from tint where a=1",
+      "explain select * from tint where a=4",
+      "explain select * from tint where a=8",
+      "explain select * from tdouble where a=1",
+      "explain select * from tdouble where a=4",
+      "explain select * from tdouble where a=8",
+      "explain select * from tdecimal where a=1",
+      "explain select * from tdecimal where a=4",
+      "explain select * from tdecimal where a=8",
+      "explain select * from tstring where a='1'",
+      "explain select * from tstring where a='4'",
+      "explain select * from tstring where a='8'",
+      "explain select * from tint where b=1 and c=1",
+      "explain select * from tint where b=4 and c=4",
+      "explain select * from tint where b=8 and c=8",
+      "explain select * from tdouble where b=1 and c=1",
+      "explain select * from tdouble where b=4 and c=4",
+      "explain select * from tdouble where b=8 and c=8",
+      "explain select * from tdecimal where b=1 and c=1",
+      "explain select * from tdecimal where b=4 and c=4",
+      "explain select * from tdecimal where b=8 and c=8",
+      "explain select * from tstring where b=1 and c=1",
+      "explain select * from tstring where b=4 and c=4",
+      "explain select * from tstring where b=8 and c=8",
```

Comment:
Should we use string type to avoid type conversion here and below?

Reply:
oh you are right.